modules/misc/test/tests/coupled_directional_mesh_height_interpolation/tests

```diff
@@ -5,6 +5,6 @@
     type = 'Exodiff'
     input = 'coupled_directional_mesh_height_interpolation.i'
     exodiff = 'coupled_directional_mesh_height_interpolation_out.e'
-    requirement = 'The system shall compute an aux variable that couples to some other value and modulates it by the mesh height in a direction.'
+    requirement = 'The system shall linearly scale a variable based on its position relative to the maximum and minimum coordinate in a user-specified direction.'
```

Comment:
`...based on its position in the relative to the minimum and maximum mesh coordinates in a user-specified direction`?

Reply:
Yeah, good suggestion. I'll make that change.